code/modules/antagonists/disease/disease_mob.dm

```diff
@@ -109,6 +111,16 @@ the new instance inside the host to be updated to the template's stats.
 			. += "<span class='notice'>[ability.name]</span>"
 
 /mob/camera/disease/say(message, bubble_type, list/spans = list(), sanitize = TRUE, datum/language/language = null, ignore_spam = FALSE, forced = null)
+	if(!message)
+		return
+	src.log_talk(message, LOG_SAY)
+	var/rendered = "<span class='sentientdisease'><b>[src]</b> says, \"[message]\"</span>"
+	for(var/mob/M in GLOB.mob_list)
```

Comment:
Better variable name.

Reply:
im not actually sure what to change it to so hopefully this is adequate, if not let me know